src/providers/oracle/qgsoracleprovider.cpp

```diff
@@ -75,7 +75,7 @@ QgsOracleProvider::QgsOracleProvider( QString const &uri, const ProviderOptions
   mUseEstimatedMetadata = mUri.useEstimatedMetadata();
   mIncludeGeoAttributes = mUri.hasParam( "includegeoattributes" ) ? mUri.param( "includegeoattributes" ) == "true" : false;
 
-  mConnection = QgsOracleConn::connectDb( mUri );
+  QgsOracleConn *mConnection = QgsOracleConn::connectDb( mUri );
```

Comment:
local variable shouldn't have m prefix.

Reply:
Hi @jef-n thanks for taking a look at the code. I know that local variables shouldn't have m prefix. I done it to propose some codes and to open the discussion about this solution.

Reply:
I have replace `mConnection` by `conn` in provider.